packages/kuma-gui/src/app/data-planes/data/DataplaneOverview.ts

```diff
@@ -73,6 +70,15 @@ export const DataplaneOverview = {
     const zone = tags.find((tag) => tag.label === 'kuma.io/zone')?.value
     const labels = typeof item.labels !== 'undefined' ? item.labels : {}
 
+    const { config } = Dataplane.fromObject({
+      type: 'Dataplane',
+      name: item.name,
+      mesh: item.mesh,
+      creationTime: item.creationTime,
+      modificationTime: item.modificationTime,
+      networking,
+    })
+
```

Comment:
Is this the `config` we eventually show in the code editor?

If so, if we where to add some of our own custom properties within `Dataplane.fromObject` I think that would mean that we might then mistakenly display these custom properties in the code editor.

I suppose the TLDR; is do we want to use `Dataplane.fromObject` or just fix up the properties that are missing using a bit of:

```js
{
  ...item.dataplane,
  name: item.name,
  ...etc
}
```

Am I making sense? Let me know if not!

Reply:
Using `Dataplane.fromObject` and only taking the `config` entry makes sure that we always show that same stuff in the config. We already show the config in the tab in the details view 🤔